Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Registration.hs

```diff
@@ -182,9 +182,9 @@ createDriverDetails personId merchantId = do
             autoPayStatus = Nothing,
             referralCode = Nothing,
             lastEnabledOn = Nothing,
-            canDowngradeToSedan = False,
-            canDowngradeToHatchback = False,
-            canDowngradeToTaxi = False,
+            canDowngradeToSedan = True,
```

Comment:
This will be default to all the merchants, should we have a TransporterConfig here? so that merchant wise we can have control on default values?

Reply:
Yes we can have that , but having vehicle varient in configs should like In future if we increase varient we have to increase configs also . We can have one default value  in configs.